### PR TITLE
[ADD] Migrated business_requirement_deliverable_categ module from v8 to v9

### PR DESCRIPTION
**Pull request by sudhir-serpentcs**
## WIP until #25  and #26  is merged

Migrated business_requirement_deliverable_categ module from v8 to v9.
@dreispt @pedrobaeza @jbeficent @moylop260 Please review.

### CODE REVIEW

File: business_requirement_deliverable_categ/README.rst [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+   :alt: License: AGPL-3
+
+==============================================
+Business Requirement Resources Task Categories
+==============================================
+
+Introduction
+============
+
+This module is part of a set of modules (`Business Requirements <https://github.com/OCA/business-requirement/blob/8.0/README.md>`_).
```
Comment (elicoidal) [inline business_requirement_deliverable_categ/README.rst]: 9.0

File: business_requirement_deliverable_categ/models/business_requirement_deliverable_categ.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import api, fields, models
+
+
+class BusinessRequirementDeliverableCateg(models.Model):
+    _inherit = "business.requirement.resource"
+
+    task_categ_id = fields.Many2one('project.tags',
+                                    string="Root Category for Tasks")
```
Comment (elicoidal) [inline business_requirement_deliverable_categ/models/business_requirement_deliverable_categ.py]: Category

**Comment by elicoidal** [comment]
:+1:  (Waiting for #25)

**Comment by elicoidal** [comment]
:+1:

**Comment by elicoidal** [comment]
@sudhir-serpentcs can you add travis information to this repo?
Any reason why Runbot is failing?

Other than that, we could be merging this PR: @astirpe @pedrobaeza @dreispt @mmalorni

**Comment by sudhir-serpentcs** [comment]
Added travis and rebased the PR.
@dreispt @pedrobaeza @jbeficent @moylop260 @elicoidal

**Review by dreispt — APPROVED** [review]

**Review by elicoidal — APPROVED** [review]

**Review by astirpe — APPROVED** [review]

**Pull request merged**